game/scripts/npc/abilities/ursa_overpower.txt

```diff
@@ -22,7 +22,7 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "15 13 11 10 9 8" //OAA
+    "AbilityCooldown"                                     "15 13 11 9 8 8"
     "AbilityDuration"                                     "15" //OAA
```

Comment:
Can the ability duration be set back to 20 seconds. This nerf makes little sense. People have been saying otherwise but I and convince ursa is trash tier. His numbers don't don't add up to this hero being good. Old Ursa >>> New Ursa and old ursa wasn't amazing (he was good but he had weird power spikes, making him hard to put on a team).

Reply:
New Ursa is just bad against bosses. New Ursa is better than old Ursa in fights. He doesnt care about disarms and other debuffs anymore. I think you are the only one that wants Ursa buffs.